admin/btop/patches/001-use-utf-force-default.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/btop.cpp
++++ b/src/btop.cpp
```

Comment:
This patch should be upstreamed, or at least there should be a pull request to upstream, where they reject it. I don't prefer to have patches in this repository, which we are going to carry forever. Also, when the patch is sent upstream, it will at least have the basic formal things like who is the author of the change, the commit subject, the commit description, etc.

Reply:
> This patch should be upstreamed, or at least there should be a pull request to upstream, where they reject it. I don't prefer to have patches in this repository, which we are going to carry forever.

This is a openwrt related issue, there are also lots of openwrt-specific hacks in tree, so do you mean we should send all of them to upstream and wait for the rejection or complain?

> Also, when the patch is sent upstream, it will at least have the basic formal things like who is the author of the change, the commit subject, the commit description, etc.

`git format-patch` is enough then.

Reply:
There are many hacks in this repository, that's true, but why should we have more? This package does not have any patches. Why should we change it?

Reply:
So you guess why we added this patch